src/Altinn.AccessManagement.Integration/Clients/PartiesClient.cs

```diff
@@ -131,6 +131,11 @@ public async Task<int> GetPartyId(string id)
         public async Task<List<Party>> GetPartiesAsync(List<int> parties)
         {
             List<Party> filteredList = new List<Party>();
+            var options = new JsonSerializerOptions
```

Comment:
There is already a `private readonly JsonSerializerOptions options` that can be used in stead of making a new one